src/components/material/MaterialDescription.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { WorkMediumFragment } from "../../core/dbc-gateway/generated/graphql";
+import { useText } from "../../core/utils/text";
+import HorizontalTermLine from "../horizontal-term-line/HorizontalTermLine";
+
+interface MaterialDescriptionProps {
+  work: WorkMediumFragment;
+  description?: string;
+}
+
+const MaterialDescription: React.FC<MaterialDescriptionProps> = ({
+  work,
+  description
+}) => {
+  const t = useText();
+
+  const seriesMembersList = work.seriesMembers.map(
+    (item) => item.titles.main[0]
+  );
+  const subjectsList = work.subjects.all.map((item) => item.display);
+
+  return (
+    <section className="material-description">
+      <h2 className="text-header-h4 pb-24">Beskrivelse</h2>
+      <p className="text-body-large material-description__content">
+        {description}
+      </p>
+      <div className="material-description__links mt-32">
+        <HorizontalTermLine
+          title={`Nr. ${work?.series[0]?.numberInSeries?.number?.[0]}`}
```

Comment:
You have to look into the schema to see what you can expect form data. There is no need for checking every single property.
Also I think it would be prettier to:
* Create a component that is called something like: SeriesLine.
* SeriesLine accepts one prop called: series
* The SeriesLine component destruct the needed parameters like:
```tsx
const SeriesLine: React.FC<SeriesLineProps> = ({
  numberInSeries: { number, display: linkList }
}) => {
``